### PR TITLE
BULQ-1318 Updates for signature tracking in UPS

### PR DESCRIPTION
**Pull request by kwoodard22**

### CODE REVIEW

File: lib/active_shipping/shipping/carriers/ups.rb [hunk 1]
```diff
@@ -109,6 +109,13 @@ class UPS < Carrier
         'M'  => 'OC'
       }
 
+      PACKAGE_DELIVERY_CONFIRMATION_CODES = {
```
Comment (kwoodard22) [inline lib/active_shipping/shipping/carriers/ups.rb]: Copied these over from Shopify/active_shipping

**Comment by jec006** [comment]
I think this looks good, @nevern02 can you take a quick look too?

**Comment by nevern02** [comment]
:+1:

**Comment by jec006** [comment]
Closing in favor of #46

**Pull request closed without merging**